plugins/catalog-react/src/hooks/useEntity.tsx

```diff
@@ -66,7 +67,13 @@ export const AsyncEntityProvider = ({
   // consumers might be doing things like `useContext(EntityContext)`
   return (
     <NewEntityContext.Provider value={createVersionedValueMap({ 1: value })}>
-      {children}
+      <AnalyticsContext
+        attributes={{
+          ...(entity ? { entityRef: stringifyEntityRef(entity) } : undefined),
```

Comment:
just a note that this attributes object isn't memo'd, but that probably doesn't matter much here